pkg/terraform/job.go

```diff
@@ -18,7 +18,8 @@ var (
 // StartJob starts a Kubernetes Job that runs Terraform on a given set of files
 // +kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;watch;create;update;patch;delete
 func StartJob(name string, namespace string, tfCmd string, ws *terraformv1alpha1.Workspace) *batchv1.Job {
-	terraformCommand := fmt.Sprintf(tfCmd, ws.Spec.WorkingDir, ws.Name)
+	workDir := fmt.Sprintf("/opt/data/%s", ws.Spec.WorkingDir)
```

Comment:
Is this defaulting the base part of the path to be `/opt/data/`? Then would the user just put something like `network` as the value for their `ws.Spec.WorkingDir`?

Is that dependent on the `image` used at all?

Reply:
Yes, this is based on the image being used. I was trying to abstract that away from the user, but that's something that the user will know, since they will build their own image. I'll revert this part back.